modules/storage-persistent-storage-pvc-volumeattributesclass.adoc

```diff
@@ -0,0 +1,67 @@
+// Module included in the following assemblies:
+//
+// * storage/understanding-persistent-storage.adoc
+//
+
+:_mod-docs-content-type: CONCEPT 
+[id="storage-persistent-storage-pvc-volumeattributesclass_{context}"]
+= Volume Attributes Classes
+
+Volume Attributes Classes provide a way for administrators to describe "classes" of storage they offer. Different classes might correspond to different quality-of-service levels.
+
+:FeatureName: Volume Attributes Classes
+include::snippets/technology-preview.adoc[leveloffset=+1]
+
+You can only use Volume Attributes Classes in {product-title} with AWS Elastic Block Storage (EBS) Container Storage Interface (CSI).
+
+You can apply a Volume Attributes Class to a persistent volume claim (PVC). If a new Volume Attributes Class becomes available in the cluster, a user can update the PVC with the new Volume Attributes Class if desired.
+
+Volume Attribute Classes have parameters that describe volumes belonging to them. Different parameters can be accepted depending on the provisioner or the resizer. If a parameter is omitted, the default is used at volume provisioning. If a user applies the PVC with a different Volume Attributes Class with omitted parameters, the default value of the parameters might be used depending on the CSI driver implementation. For more information, see the related CSI driver documentation .
+
+There can be at most 512 parameters defined for a `VolumeAttributesClass`. The total length of the parameters object including its keys and values cannot exceed 256 KiB.
+
+== Enabling Volume Attributes Classes 
+Volume Attributes Classes are not enabled by default.
+
+To enable Volume Attributes Classes, follow the procedure in xref:../nodes/clusters/nodes-cluster-enabling-features.adoc#nodes-cluster-enabling-features[Enabling {product-title} features using FeatureGates].
+
+== Defining Volume Attributes Classes
+
+The following is an example Volume Attributes Class YAML file.
+
+.`VolumeAttributesClass` definition example
+[source,yaml]
+----
+apiVersion: storage.k8s.io/v1beta1
+kind: VolumeAttributesClass <1>
+metadata:
+  name: silver <2>
+driverName: pd.csi.storage.gke.io <3>
```

Comment:
If we only have EBS available we should not include a GCP PD example

Reply:
Good catch!